lib/storage-client/googleapis.js

```diff
@@ -95,7 +95,7 @@ export function parseNotes(content) {
  * @returns {Promise<ChromedriverDetailsMapping>}
  */
 export async function parseGoogleapiStorageXml(xml, shouldParseNotes = true) {
-  const doc = new DOMParser().parseFromString(xml);
+  const doc = new DOMParser().parseFromString(xml, 'application/xml');
```

Comment:
Could you import `MIME_TYPE` like https://github.com/appium/appium/pull/20513 instead of giving vanilla value?
It is appreciated to set `^0.9` for https://github.com/appium/appium-chromedriver/blob/master/package.json#L47 as well

Reply:
I believe we need to import the same constant as in https://github.com/appium/appium/pull/20513/files

Reply:
also it definitely makes sense to bump the minimum version of xmldom in package.json to 0.9.0